app/views/ideas/show.html.erb

```diff
@@ -1,2 +1,6 @@
-<h1>Ideas#show</h1>
-<p>Find me in app/views/ideas/show.html.erb</p>
+<h1>Idea <%=@idea.id %> </h1>
```

Comment:
Spaces <3

Reply:
@mkrajewska you can check style guidelines here https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide